raiden/tests/scenarios/scenario-funding channel.yaml

```diff
@@ -0,0 +1,256 @@
+
+      - serial:
+          name: Scenario F12 (Depositing incredibly huge number)
+          tasks:
+           - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+           - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000_000, expected_http_status: 402} ## !! the problem is, the node does not have that amount of tokens so it would return "not sufficient funds"
+           - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F13 (Depositing smallest possible number)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_000, balance: 99_000_000_000_000_000_000, state: "opened"}
+            - deposit: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, expected_http_status: 200}
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+
+#      - serial:
+#          name: Scenario F14 (Depositing more than channel limit of 100 - with ETH it would be 0.15 ETH)
+#          tasks:
+#            - deposit: {from: 0, to: 1, total_deposit: 100} ## This can not be tested with this token
+
+      - serial:
+          name: Scenario F15 (Depositing all available on-chain balance of node 1)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_000_000_000_000_000_001, balance: 99_000_000_000_000_000_001, state: "opened"}
+            - deposit: {from: 0, to: 1, total_deposit: 99_999_000_000_000_000_000, expected_http_status: 200}
+            - assert: {from: 0, to: 1, total_deposit: 99_999_000_000_000_000_000, balance: 99_999_000_000_000_000_000, state: "opened"}
+
+      - serial:
+          name: Scenario F16 (Depositing more than available on-chain balance of node 1)
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_999_000_000_000_000_000, balance: 99_999_000_000_000_000_000, state: "opened"}
+            - deposit: {from: 0, to: 1, total_deposit: 100_99_999_000_000_000_000_001, expected_http_status: 402} ## include more than available on chain balance and error code
+            - assert: {from: 0, to: 1, total_deposit: 99_999_000_000_000_000_000, balance: 99_999_000_000_000_000_000, state: "opened"}
+
+      - parallel:
+          name: Scenario F20 (Deposit and then kill node before the transaction was submitted to the blockchain) #IS THIS A USEFUL TEST? DOES THIS TEST WHAT IS DESCRIBED?
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 99_999_000_000_000_000_000, balance: 99_999_000_000_000_000_000, state: "opened"}
```

Comment:
the assert and deposit should be in a serial sub-task as guess. We want to first assert, then depoit.